data/json/recipes/food/bread.json

```
@@ -33,17 +33,18 @@
     "skill_used": "cooking",
     "difficulty": 3,
     "charges": 10,
-    "time": "30 m",
+    "time": "1 h 30 m",
     "batch_time_factors": [ 50, 5 ],
+    "autolearn": false,
     "book_learn": [ [ "family_cookbook", 1 ], [ "baking_book", 2 ], [ "cookbook", 2 ], [ "cookbook_daintydishes", 2 ] ],
     "qualities": [ { "id": "COOK", "level": 3 } ],
     "tools": [ [ [ "surface_heat", 8, "LIST" ] ] ],
     "components": [
       [ [ "bread_flour", 1 ] ],
-      [ [ "flour", 1 ] ],
-      [ [ "cornmeal", 1 ] ],
-      [ [ "molasses", 1 ] ],
-      [ [ "water", 1 ] ],
+      [ [ "flour", 5 ] ],
+      [ [ "cornmeal", 5 ] ],
+      [ [ "molasses", 2 ] ],
+      [ [ "water", 1 ], [ "water_clean", 1 ] ],
```

```suggestion
      [ [ "water_clean", 1 ] ],
```
I'm wondering if unclean water should ever be used for baking bread?  Someone who knows more should definitely weigh in on this question.

---

It would really depend on exactly how unclean the water is. If it's just stuff that would be killed by heat, you'd be fine. If there were things floating in it, that'd be another matter entirely.

---

Yeah this would go back into that issue put up by kevingranade talking about water purity. Personally, I think it's good as it is until we do a full revamp of determining how clean water is.